testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -1101,6 +1101,8 @@ public void testCreateSchema()
         assertThat(computeActual("SHOW SCHEMAS").getOnlyColumnAsSet()).doesNotContain(schemaName);
         assertUpdate("CREATE SCHEMA " + schemaName);
         assertThat(computeActual("SHOW SCHEMAS").getOnlyColumnAsSet()).contains(schemaName);
+        assertThat((String) computeScalar("SHOW CREATE SCHEMA " + schemaName))
```

Comment:
Maybe add a comment about the intent (verify SHOW CREATE SCHEMA is works if CREATE SCHEMA works?) because it's not obvious.

Reply:
that's one of the assertions of a test. to me, it's as related to the test as eg checking `SHOW SCHEMAS`.
did i convince you?